docs/contributing/coding-guidelines/index.md

```diff
@@ -14,6 +14,58 @@ Also, keep in mind the following concepts.
 
 - Keep things consistent.
 - Automate where possible, using simple checks for formatting, syntax, etc.
+- Write comments and documentation in English.
+- Functions that are too complex (low cohesion) should be appropriately split into smaller functions (e.g. less than 40 lines in one function is recommended in the [google style guideline](https://google.github.io/styleguide/cppguide.html#Write_Short_Functions)).
+- Try to minimize the use of member variables or global variables that have a large scope.
+- Whenever possible, break large pull requests into smaller, manageable PRs (less than 200 lines of change is recommended in some research e.g. [here](https://opensource.com/article/18/6/anatomy-perfect-pull-request)).
 - When it comes to code reviews, don't spend too much time on trivial disagreements. For details see:
   - <https://en.wikipedia.org/wiki/Law_of_triviality>
   - <https://steemit.com/programming/@emrebeyler/code-reviews-and-parkinson-s-law-of-triviality>
+
+## C++ Style Guide
```

Comment:
Since C++ and ROS have their own individual pages, I think it's easy to overlook the information written in the index.

Reply:
Thanks! 👍 
Now I'm changing it following this comment!

https://github.com/autowarefoundation/autoware-documentation/pull/599#issuecomment-2297869052